modules/plugins/languages/rust.nix

```diff
@@ -102,7 +102,7 @@ in {
         startPlugins = ["crates-nvim"];
         lsp.null-ls.enable = mkIf cfg.crates.codeActions true;
         autocomplete.nvim-cmp.sources = {crates = "[Crates]";};
-        pluginRC.rust-crates = entryAnywhere ''
+        pluginRC.rust-crates = entryAfter ["lsp-setup"] ''
```

Comment:
You don't have to change it here, revert

Reply:
Reverted